testutils/store_gossiper/store_gossiper.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Matt Tracy (matt@cockroachlabs.com)
+// 		   Bram Gruneir (bram+code@cockroachlabs.com)
+
+package storeGossiper
```

Comment:
Can you break the cycle a different way?

On Tue, Sep 8, 2015 at 9:02 PM, Bram Gruneir <notifications@github.com>
wrote:

> In testutils/store_gossiper/store_gossiper.go
> <https://github.com/cockroachdb/cockroach/pull/2336#discussion_r38998292>:
>
> > +// you may not use this file except in compliance with the License.
> > +// You may obtain a copy of the License at
> > +//
> > +//     http://www.apache.org/licenses/LICENSE-2.0
> > +//
> > +// Unless required by applicable law or agreed to in writing, software
> > +// distributed under the License is distributed on an "AS IS" BASIS,
> > +// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
> > +// implied. See the License for the specific language governing
> > +// permissions and limitations under the License. See the AUTHORS file
> > +// for names of contributors.
> > +//
> > +// Author: Matt Tracy (matt@cockroachlabs.com)
> > +// 		   Bram Gruneir (bram+code@cockroachlabs.com)
> > +
> > +package storeGossiper
>
> I'm not sure what the convention is. I went looking and didn't see
> anything.
>
> sadly, testutils created a import cycle.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2336/files#r38998292>.
>